string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,13 @@
+/* exported isLowerCased */
+/* Use toLowerCase() on word and store lowercased word
+   Compare lowercased word with word and see if they are the same
+   If they are the same return true, if not return false */
+function isLowerCased(word) {
+  var lowerCasedWord;
```

Comment:
This is not necessary in JavaScript, you can assign a value as you declare a variable.